### DISCUSSION TITLE
regarding command issue

### DISCUSSION THREAD

**Discussion by anujyadav0693** [neperfepx/neper#684] [Q&A] [answered]
respected sir,
i am using neper version 4.1.2  in HPC and neper version 4.4.2 in PC.
in version 4.4.2 command -ori "fiber(0,0,1,0,0,1) is working but in version 4.1.2 not working.
should i update to the version to use this in hpc.

**Comment by rquey** [comment, marked as answer]
Version 4.1.2 is 2+-year old, so, yes, it is a good idea to update. Current version is 4.6.0.

**Reply by anujyadav0693** [reply]
thank you for your valuable reply.
but sir, I am trying to install the latest version in conda environment. 
i have tried with conda but not available
Please provide all necessary information such as weblink to the download page, installation instructions.
